### ISSUE TITLE
Create a new flag that not allow the interpretation of "inline ignores"

### ISSUE THREAD

**Issue by iaghoCristian** [hadolint/hadolint#747]
- [ ] This is a bug report
- [X] This is a feature request
- [X] I searched existing issues before opening this one

### Expected behavior
Possibility ignore the "inline hadolint ignores".
Create a flag that allows full interpretation of the Dockerfile, even if there are "inline ignores".

### Actual behavior
When using hadolint to scan Dockerfile is possible to bypass warnings by using "hadolint ignores" instead of actually fix the errors/warnings found in the Dockerfile

### Feature request specification:
**_--discard-inline-ignores_**  flag that does not allow the interpretation of "inline ignores"
e.g: ``` hadolint --discard-inline-ignores Dockerfile ```

**Comment by felipecrs** [comment]
This is an interesting feature for us. We have a centrally managed hadolint configuration file which we use to lint Dockerfiles across many repositories, and we also don't want developers to ignore rules by their own.

Apart from the cli flag, it would also be cool if it could be set in the configuration file perhaps, like:

```yaml
failure-threshold: string
```

**Comment by felipecrs** [comment]
OMG, this is brilliant! Thank you a lot for this, @m-ildefons!
